### DISCUSSION TITLE
How to Achieve Redis HA in OriginMapStore

### DISCUSSION THREAD

**Discussion by UOSCS** [OvenMediaLabs/OvenMediaEngine#1722] [Help]
Hello,

Does OriginMapStore support Redis Sentinel or Redis Cluster?

I’ve built an origin-edge cluster using OriginMapStore. 
Since the OriginMapStore architecture relies on Redis, if there's a failure in the Redis server, video transmission and viewing become impossible. 
I found that when I entered the Redis Sentinel address in the OriginMapStore Redis host, it does not seem to support Redis Sentinel.

Is there any method suggested by OME to achieve Redis high availability?

**Comment by getroot** [comment]
I don't have any experience with Redis HA yet. Is there anything I need to do on the Redis Client side (OvenMediaEngine)? If you have any related information, please share it.
